### PR TITLE
Run wideband+narrowband in test_engine

### PR DESCRIPTION
**Pull request by bmerry**
Instead of only running the desired output stream (which does not test the real-world use of running both concurrently), run both and only do tests against one at a time.

The main functional change in the test is that previously --dst-chunk-jones was set differently for wide and narrow, but now they need to have a single value. I kept the value picked for narrowband just because that made all the tests pass.

Closes NGC-955.

Checklist (if not applicable, edit to add `(N/A)` and mark as done):

- [x] (N/A) If dependencies are added/removed: update `setup.cfg` and `.pre-commit-config.yaml`
- [x] (N/A) If modules are added/removed: use sphinx-apidoc to update files in `doc/`
- [x] Ensure copyright notices are present and up-to-date
- [x] (N/A) If qualification tests are changed: attach a sample qualification report
- [x] (N/A) If design has changed: ensure documentation is up to date
- [x] (N/A) If ICD-defined sensors have been added: update `fake_servers.py` in katsdpcontroller to match

### CODE REVIEW

File: test/fgpu/test_engine.py [hunk 1]
```diff
@@ -47,18 +48,23 @@
...
-CHUNK_JONES = 1048576
+CHUNK_JONES = 131072
 MAX_DELAY_DIFF = 16384  # Needs to be lowered because CHUNK_SAMPLES is lowered
 PACKET_SAMPLES = 4096
 TAPS = 16
 FENG_ID = 42
 ADC_SAMPLE_RATE = 1712e6
 
+WIDEBAND_ARGS = f"name=test_wideband,dst=239.10.11.0+15:7149,taps={TAPS}"
+# Centre frequency is a multiple of the channel width
```
Comment (james-smith-za) [inline test/fgpu/test_engine.py:63]: Is that true regardless of the number of channels?

File: test/fgpu/test_engine.py [hunk 2]
```diff
@@ -112,30 +118,31 @@ def frac_channel(output: Output, channel: float) -> float:
...
+        return f"{NARROWBAND_ARGS},channels={channels}"
+
+    @pytest.fixture(params=["wideband", "narrowband"])
+    def output(self, wideband_args: str, narrowband_args: str, request: pytest.FixtureRequest) -> Output:
+        """The output to run tests against."""
+        if request.param == "wideband":
+            return parse_wideband(wideband_args)
         else:
-            return parse_narrowband(arg_value)
+            return parse_narrowband(narrowband_args)
```
Comment (james-smith-za) [inline test/fgpu/test_engine.py:137]: So if I'm understanding correctly, this test is more or less the same as it previously was, except that
- it's now repeated twice,
- now both wide and narrowband are running, but
- only one output is checked each time.

Do I have it more or less correct?

File: test/fgpu/test_engine.py (hunk 1 above)
Comment (bmerry) [reply test/fgpu/test_engine.py:63]: Probably not - just for the specific CHANNELS constant defined here.

File: test/fgpu/test_engine.py (hunk 2 above)
Comment (bmerry) [reply test/fgpu/test_engine.py:137]: Yes.

**Review by james-smith-za — APPROVED** [review]

**Review by amishatishpatel — APPROVED** [review]

**Pull request merged**